discord/utils.py

```diff
@@ -554,6 +555,29 @@ def utcnow() -> datetime.datetime:
     return datetime.datetime.now(datetime.timezone.utc)
 
 
+def to_time_mention(time: datetime.datetime, mention_type: Optional[TimeMentionType] = None) -> str:
+    """A helper function to make a time mention from ``datetime.datetime``.
+
+    .. versionadded:: 2.0
+
+    Parameters
+    -----------
+    time: :class:`datetime.datetime`
+        The ``datetime`` to make mention.
+    mention_type: :class:`TimeMentionType`
+        Type of the mention.
+
+    Returns
+    --------
+    :class:`str`
+        A datetime mention for given datetime.
+    """
+    if mention_type is None or mention_type is TimeMentionType.default:
+        return f"<t:{int(time.timestamp())}>"
+    else:
+        return f"<t:{int(time.timestamp())}:{mention_type.value}>"
+
+
```

Comment:
Same as above, unsure about the naming, `to_timestamp_format` might be a better suited name?

Reply:
I changed to `to_timestamp_format`, thanks